### ISSUE TITLE
Default PNG frame delay might be 120ms and not 100ms

### ISSUE THREAD

**Issue by PirateIzzy** [Kagamia/WzComparerR2#301]
From my observations in-game, whenever a delay is not specified, it should be 120ms, rather than 100ms. The example I found is Amy (NPC ID 9400920). Her walk animation doesn't specify animation delays.
![unknown](https://github.com/user-attachments/assets/16c6b1a2-c8e3-491f-a165-8f532a9d16e5)

This is how her speed would be at 100ms per frame, 8 frames (800ms total)
![9400920 img move](https://github.com/user-attachments/assets/85fc13d9-cc57-439e-a0f6-aa52a62ba3ff)

But the animation is slightly slower in game.
https://github.com/user-attachments/assets/4a008d2e-ad5c-4c16-8738-81a1463e633e

I reduced the speed to 83%, so it's approximately 120ms, and it looks a lot more accurate to in-game:
![ezgif-4-937441d4a0](https://github.com/user-attachments/assets/ad4c6006-f7e6-4954-8199-75a03ca7f567)

This may require more investigation, as Nexon may have defined it differently based on the scenario. But I definitely think we should keep an eye on this for more examples.

**Comment by Kagamia** [comment]
Yes it makes sense. Most skill animations and actions have frame intervals that are multiples of 30ms.

**Comment by PirateIzzy** [comment]
Confirming that it's fixed.

**Comment by PirateIzzy** [comment]
I am reopening this issue because I noticed that under Avatar, default delays are still set to 100ms instead of 120ms. I tested this by uploading a chair on the MapleStory Wiki as APNG with both Item -> Effect, and Avatar -> Mount.

Von Bon's Von Chair (Item ID 3010512), Item -> Effect:
![image](https://github.com/user-attachments/assets/05bbd1d4-0e57-4eea-b76e-0e5dc1d55922)

Avatar -> Mount:
![image-1](https://github.com/user-attachments/assets/7aa2ac15-782c-4823-9650-9fce1a3a7b75)

**Comment by Kagamia** [comment]
Applied 1-line fix.
Please let me know if I missed anything else.

**Comment by PirateIzzy** [comment]
![image](https://github.com/user-attachments/assets/768d0d79-432d-4f31-9d90-4512600fd064)
The only other fields I see are these two, which might also have the same problem.

**Comment by Kagamia** [comment]
`MultiFrameAnimationData` does not exist in master branch, and the code in `TextureLoader` has already obsoleted that we can ignore it.

**Comment by PirateIzzy** [comment]
Understood, in that case we're all set. Thank you 👍
